devfile.yaml

```diff
@@ -0,0 +1,37 @@
+schemaVersion: 2.2.0
+metadata:
+  name: fleet-manager
+  version: 0.1.0
+  provider: Red Hat
+  icon: https://design.jboss.org/quarkus/logo/final/SVG/quarkus_icon_rgb_default.svg
+  displayName: Fleet Manager
+  description: Service Registry Fleet Manager
+  tags:
+    - Java
+    - Apicurio
+  projectType: Quarkus
+  language: Java
+  attributes:
+    alpha.dockerimage-port: 8081
+parent:
+  id: java-quarkus
+  registryUrl: 'https://registry.devfile.io'
+components:
+  - name: image-build
+    image:
+      imageName: java-quarkus-image:latest
```

Comment:
FM is currently based on Quarkus `2.7.5.Final`, will using the `latest` image cause any issues?

Reply:
My bad, this represents the name of the image that we build. Updated with the correct image name